packages/block-library/src/gallery/block.json

```diff
@@ -48,6 +48,11 @@
 		"columns": {
 			"type": "number"
 		},
+		"galleryCaption": {
```

Comment:
Can it be `caption`? It seems repetitive to include gallery as a prefix.

Reply:
The problem is that gallery images already have `caption`s so it's less confusing if this one has a name that clearly indicates what it is a caption of.

Reply:
It depends on where you look at, I found it confusing when seeing it in the `attributes` section `block.json` file. Gallery itself can have only one caption. In the future, we might refactor this block and convert all images to nested blocks at the issue you pointed out would solve on its own. You can always alias the caption of an image if it has to be referenced in the same scope or extract the image logic to its own component.